README.md

```diff
@@ -43,7 +48,9 @@ Acquiring tokens with MSAL Python follows this 3-step pattern.
 
    ```python
    from msal import PublicClientApplication
-   app = PublicClientApplication("your_client_id", authority="...")
+   app = PublicClientApplication(
+       "your_client_id",
+       "authority": "https://login.microsoftonline.com/Enter_the_Tenant_Name_Here")
```

Comment:
Suggestion: tenant-id would be more useful as tenant name can also mean tenant`s actual name  instead of xyz.onmicrosoft.com

Reply:
You might have a point. I don't necessarily know our official terminology here, I was just copying that entire phrase from our existing samples [here](https://github.com/Azure-Samples/ms-identity-python-webapp/blob/master/app_config.py#L12) and [there](https://github.com/Azure-Samples/ms-identity-python-daemon/blob/master/1-Call-MsGraph-WithSecret/parameters.json#L2), which have presumably been vetted by our PMs and doc writers.

I would say let's use this term as-is for now. You/we can bring this up as a separated suggestion, if desirable.